charts/aspenmesh-collector/templates/configmap.yaml

```diff
@@ -62,6 +62,19 @@ data:
                 action: keep
                 regex: istiod;http-monitoring
     processors:
+      metricstransform:
+        transforms:
+          # for all kube-state-metrics metrics, rename the labels app and version to app.kubernetes.io/name and app.kubernetes.io/version
+          - include: ^kube_.*$
+            match_type: regexp
+            action: update
+            operations:   
+              - action: update_label
+                label: label_app
+                new_label: label_app_kubernetes_io_name
+              - action: update_label
+                label: label_version
+                new_label: label_app_kubernetes_io_version
```

Comment:
I almost like the simpler `label_app`/`label_version` label names better than the `label_app_kubernetes_io` prefixed ones. I almost wonder if we should flip the normalization, or even rename both to `app` and `version` to keep things simpler.

Reply:
Will the current `metricLabelsAllowlist` be able to get all metrics with `app` and `version` labels? Can you explain more why we want to rename the labels here?

Reply:
`app` & `version` are Istio-specific. Sticking with [Kubernetes' best practices](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) for labels will allow for other meshes to be adopted later on.

Reply:
@winsock - where are you seeing the `label_*`? It should just be `app` or `version` as an example.

Reply:
@ricksalsa `kube-state-metrics` prefixes it with `label_` documented here(kube_deployment_labels) https://github.com/kubernetes/kube-state-metrics/blob/master/docs/deployment-metrics.md

Same applies for the pod version

Reply:
Thanks, @winsock, I found it any `kube_*_labels` in my local deployment. Moving `label_[app|version]` to `label_app_kubernetes_io_[app|version]` makes sense here.

Reply:
Updated PR if you want to include this